### PR TITLE
use assert instead of if-panic

### PR DESCRIPTION
**Pull request by mrmiywj**
#727

---
This change is [Reviewable](https://reviewable.io/reviews/servo/webrender/1023)

### CODE REVIEW

File: wrench/src/yaml_frame_reader.rs [hunk 1]
```diff
@@ -491,9 +487,7 @@ impl YamlFrameReader {
                                      .expect("Text items with glyphs require bounds [for now]");
             (glyphs, rect)
         } else {
-            if native_key.is_none() {
-                panic!("Can't layout simple ascii text with raw font [for now]");
-            }
+            assert!(!native_key.is_none(), "Can't layout simple ascii text with raw font [for now]");
```
Comment (kvark) [inline wrench/src/yaml_frame_reader.rs]: nit: could use `is_some()`

**Comment by mrmiywj** [comment]
PTAL @kvark

**Comment by kvark** [comment]
@mrmiywj please use `assert_eq!`

**Comment by glennw** [comment]
@mrmiywj Could you squash those commits? Then this should be good to go :)

**Comment by mrmiywj** [comment]
@glennw PTAL

**Comment by kvark** [comment]
Thanks, looks great!
@bors-servo r+

**Pull request merged**